### PR TITLE
add find vg name by lv volume id func

### PR DESCRIPTION
**Pull request by leanenkaa**

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -248,3 +248,34 @@ func RemoveTagLV(ctx context.Context, vg string, name string, tags []string) (st
...
+
+// FindVGbyLV VG name in which particular LV exists
+func FindVGbyLV(ctx context.Context, volId string) (vgName *string, err error) {
+	sp, _ := opentracing.StartSpanFromContext(ctx, "lvm.findvgbylv")
+	sp.SetTag("component", "lvm")
+	sp.SetTag("span.kind", "client")
+	defer sp.Finish()
+
+	cmd := exec.Command("lvs", "--units=b", "--separator=<:SEP:>", "--nosuffix", "--noheadings",
+		"-o", "lv_name,lv_size,lv_uuid,lv_attr,copy_percent,lv_kernel_major,lv_kernel_minor,lv_tags, vg_name", "--nameprefixes", "-a")
```
Comment (farcaller) [inline commands/commands.go]: can you please remove the extra space in `lv_tags, vg_name`?

**Review by farcaller — CHANGES_REQUESTED** [review]

**Comment by leanenkaa** [comment]
@googlebot I signed it!

**Comment by leanenkaa** [comment]
@googlebot I signed it!

**Comment by farcaller** [comment]
Your first commit in the chain is authored by `aleanenka@evolutiongaming.com` which didn't sign the CLA, according to the bot. You'll need to either sign the CLA with that email too or squash your commits together.

**Comment by leanenkaa** [comment]
@googlebot I signed it!

**Comment by farcaller** [comment]
So I'm making a closer pass on this and I have some questions. I'm not entirely convinced `FindVGbyLV` is needed at all: practically speaking, you can filter the ListLV output, and you're not dealing with anything but `lvs` in here, so it doesn't warrant a new `command`, really. The processing happens higher the stack.

Then, you return the `volume_group` as a name string, instead of the `VolumeGroup` message.  This makes the use case extremely narrow: e.g. if I wanted to query the size of the VG I'd have to then `ListVG` and match by name in the end user code either way.

The addition of `VGName` is a good point, but maybe everything else should exist in the client code—e.g. the filtering. That results in more bytes over the wire, but again, from the practical standpoint, it shouldn't be a major issue.

Is there a good reason to filter this server-side that I'm missing? Please advise.

**Comment by leanenkaa** [comment]
I thought about how to deal with my case with minimal impact for the library but had no idea and decided to add another function.
The case is that I need to find volume group name by logical volume name that exists in this VG, so the only info that I have is logical volume name and I agree that it will be more convenient to use `ListLV` func for this but I don't have any `listspec` to pass to this func.
I'll appreciate if you have any idea how to solve this another way.

**Comment by leanenkaa** [comment]
It seems that I can pass empty `listspec` to `ListLV` func and then I can iterate and filter `LV` struct list and find the appropriate VG. I'll try this but I still need `VGName` in `LV` struct.

**Comment by farcaller** [comment]
> It seems that I can pass empty listspec to ListLV func and then I can iterate and filter LV struct list and find the appropriate VG. I'll try this but I still need VGName in LV struct.

This would be my solution as well. Would you mind simplifying your PR so it only contains the relevant change then?

**Comment by leanenkaa** [comment]
Of course, will do.

**Review by farcaller — CHANGES_REQUESTED** [review]
This will still require an additional update to the proto structure and the `LV.ToProto()` to make this field available to the client.

File: proto/lvm.proto [hunk 2]
```diff
@@ -102,6 +102,7 @@ message LogicalVolume {
   uint32 actual_dev_major_number = 6;
   uint32 actual_dev_minor_number = 7;
   repeated string tags = 8;
+  string VGName = 9;
```
Comment (farcaller) [inline proto/lvm.proto]: This will be `vg_name` to follow the style of this proto.

**Comment by farcaller** [comment]
Thanks for your work; just one last thing there.

**Comment by leanenkaa** [comment]
Thank you for your patience and help :)

**Comment by farcaller** [comment]
👍

**Pull request merged**